angular.json

```diff
@@ -0,0 +1,139 @@
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "version": 1,
+  "newProjectRoot": "projects",
+  "projects": {
+    "contact-picker-widget-angular": {
+      "root": "",
+      "sourceRoot": "example",
+      "projectType": "application",
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:browser",
+          "options": {
+            "outputPath": "dist",
+            "index": "example/index.html",
+            "main": "example/main.ts",
+            "tsConfig": "example/tsconfig.app.json",
+            "polyfills": "example/polyfills.ts",
+            "assets": [
+              "example/assets",
+              "example/favicon.ico"
+            ],
+            "styles": [
+              "example/styles.scss"
+            ],
+            "scripts": []
+          },
+          "configurations": {
+            "prod": {
+              "fileReplacements": [
+                {
+                  "replace": "example/config/environment.ts",
+                  "with": "example/config/environment.prod.ts"
+                }
+              ]
+            },
+            "production": {
+              "optimization": true,
+              "outputHashing": "all",
+              "sourceMap": false,
+              "extractCss": true,
+              "namedChunks": false,
+              "aot": true,
+              "extractLicenses": true,
+              "vendorChunk": false,
+              "buildOptimizer": true
+            }
+          }
+        },
+        "serve": {
+          "builder": "@angular-devkit/build-angular:dev-server",
+          "options": {
+            "browserTarget": "contact-picker-widget-angular:build",
```

Comment:
Kan je hier de naam van de repo behouden (zijnde `contact-picker_widget_angular `)? (ook in de rest van de file trouwens)